### PR TITLE
Sprint1 Story 7 emit websocket for users to redirect to swiping page and add swiping container test

### PR DESCRIPTION
**Pull request by rosakng**

### CODE REVIEW

**Review by jeffreyquiambao — APPROVED** [review]
Great job! looks good

File: wewatch-client/src/components/Lobby/Lobby.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import queryString from 'query-string';
 import io from "socket.io-client";
-import { Link } from "react-router-dom";
```
Comment (dlhxu) [inline wewatch-client/src/components/Lobby/Lobby.js:4]: Good catch removing the unused Link import. This keeps our code clean and understandable in the future!

**Review by dlhxu — APPROVED** [review]
Looks good!

**Review by varoongupta — APPROVED** [review]
looks great!

**Review by jeffreyquiambao — APPROVED** [review]
lgtm!

File: wewatch-client/src/components/Lobby/Lobby.js (hunk 1 above)
Comment (rosakng) [reply wewatch-client/src/components/Lobby/Lobby.js:4]: Thanks derek! great work helping me with that websocket bug! 🚀

**Review by billxsheng — APPROVED** [review]

**Pull request merged**